contracts/optimistic-ethereum/OVM/accounts/OVM_ECDSAContractAccount.sol

```diff
@@ -76,6 +79,12 @@ contract OVM_ECDSAContractAccount is iOVM_ECDSAContractAccount {
             "Transaction nonce does not match the expected nonce."
         );
 
+        // Need to make sure that the gas is sufficient to execute the transaction.
+        Lib_SafeExecutionManagerWrapper.safeREQUIRE(
+           gasleft() >= SafeMath.add(decodedTx.gasLimit, EXECUTION_VALIDATION_GAS_OVERHEAD),
```

Comment:
Is the openzeppelin safeMath ok to use here? I think that will call `require` directly instead of `safeREQUIRE`?

Reply:
yes you are right...

Reply:
solved instead with Lib_SafeMathWrapper